### PR TITLE
Mostly Linter Things

### PR DESCRIPTION
**Pull request by olliethomas**
- mostly linter things
- balck and isort

### CODE REVIEW

File: conduit/models/self_supervised/multicrop.py [hunk 1]
```diff
@@ -51,7 +49,9 @@ def with_dino_transform(
         local_crops_scale: tuple[float, float] = (0.05, 0.4),
         local_crops_number: int = 8,
     ) -> MultiCropTransform:
-        from conduit.models.self_supervised.dino.transforms import dino_train_transform
+        from conduit.models.self_supervised.dino.transforms import (  # TODO: Cyclic Import
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/multicrop.py:52]: This doesn't matter because the import is lazy

File: conduit/models/self_supervised/multicrop.py [hunk 2]
```diff
@@ -51,7 +49,9 @@ def with_dino_transform(
...
+            dino_train_transform,
+        )
```
Comment (olliethomas) [inline conduit/models/self_supervised/multicrop.py:54]:
```suggestion
        from conduit.models.self_supervised.dino.transforms import dino_train_transform
```

File: conduit/models/self_supervised/moco/transforms.py [hunk 3]
```diff
@@ -11,31 +9,12 @@
...
 
 __all__ = [
-    "GaussianBlur",
     "moco_ft_transform",
     "moco_test_transform",
     "mocov2_train_transform",
 ]
 
-
-class GaussianBlur:
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/moco/transforms.py:21]: I don't agree with this move. While this transform is used by DINO that so happens because it inherits its transforms partly from MoCo.

File: conduit/models/self_supervised/dino/transforms.py [hunk 4]
```diff
@@ -9,11 +9,12 @@
 from conduit.constants import IMAGENET_STATS
 from conduit.data.datasets.utils import PillowTform
 from conduit.data.structures import MeanStd
-from conduit.models.self_supervised.moco import GaussianBlur
 from conduit.models.self_supervised.multicrop import MultiCropTransform
 
 __all__ = ["Solarization", "dino_train_transform"]
 
+from conduit.models.self_supervised.transforms import GaussianBlur
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/dino/transforms.py:16]: Misplaced

File: conduit/models/self_supervised/dino/loss.py [hunk 5]
```diff
@@ -107,11 +106,11 @@ def forward(
         total_loss = student_logits_seq[-1].new_zeros(())
         n_loss_terms = student_logits_seq[-1].new_zeros(())
         for iq, q in enumerate(teacher_probs_seq):
-            for v in range(len(student_logits_seq)):
+            for v, sl in enumerate(student_logits_seq):
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/dino/loss.py:109]: bit of a pointless change

File: conduit/fair/metrics/accuracy.py [hunk 6]
```diff
@@ -10,7 +10,7 @@ class FbAccuracy(Accuracy):
 
     @property
     def __name__(self) -> str:
```
Comment (MylesBartlett) [inline conduit/fair/metrics/accuracy.py:12]: This metric is no longer used by the way

File: conduit/data/datasets/vision/celeba.py [hunk 7]
```diff
@@ -10,7 +10,8 @@
 import torch
 
 from conduit.data.datasets.utils import FileInfo, ImageTform, download_from_gdrive
-from conduit.data.datasets.vision.base import CdtVisionDataset
+
+from .base import CdtVisionDataset
```
Comment (MylesBartlett) [inline conduit/data/datasets/vision/celeba.py:14]: Why make this change? The absolute path is perfectly functional and is what auto-import uses.

File: conduit/architectures/vit.py [hunk 8]
```diff
@@ -202,6 +201,8 @@ def __init__(
         norm_layer: type[nn.LayerNorm] | type[nn.BatchNorm1d] = nn.LayerNorm,
     ) -> None:
         super().__init__()
+        if isinstance(img_size, int):
```
Comment (thomkeh) [inline conduit/architectures/vit.py:204]: Hm, maybe check for "not list or tuple" instead?

File: conduit/callbacks/posthoc_eval.py [hunk 9]
```diff
@@ -22,6 +22,8 @@ class PostHocEval(pl.Callback):
         - ``self.trainer''
     """
 
+    eval_clf: FineTuner
```
Comment (MylesBartlett) [inline conduit/callbacks/posthoc_eval.py:25]: let's just delete this file

File: conduit/data/datasets/vision/celeba.py (hunk 7 above)
Comment (olliethomas) [reply conduit/data/datasets/vision/celeba.py:14]: it's clearer that you're importing from a file that's in the same module

Comment (thomkeh) [reply conduit/data/datasets/vision/celeba.py:14]: Local imports are less likely to be cyclic

File: conduit/models/self_supervised/moco/transforms.py (hunk 3 above)
Comment (olliethomas) [reply conduit/models/self_supervised/moco/transforms.py:21]: but there's nothing moco specific about this transform, and importing from another module at the same level isn't great

**Review by thomkeh — APPROVED** [review]

File: conduit/data/datasets/vision/celeba.py (hunk 7 above)
Comment (MylesBartlett) [reply conduit/data/datasets/vision/celeba.py:14]: But such imports are also more brittle in the face of refactoring

File: conduit/models/self_supervised/dino/transforms.py [hunk 10]
```diff
@@ -10,11 +10,10 @@
 from conduit.data.datasets.utils import PillowTform
 from conduit.data.structures import MeanStd
 from conduit.models.self_supervised.multicrop import MultiCropTransform
+from conduit.models.self_supervised.transforms import GaussianBlur
 
 __all__ = ["Solarization", "dino_train_transform"]
 
-from conduit.models.self_supervised.transforms import GaussianBlur
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/dino/transforms.py]: Still think this should stay in MoCo

File: conduit/models/self_supervised/dino/transforms.py [hunk 11]
```diff
@@ -10,11 +10,10 @@
 from conduit.data.datasets.utils import PillowTform
 from conduit.data.structures import MeanStd
 from conduit.models.self_supervised.multicrop import MultiCropTransform
+from conduit.models.self_supervised.transforms import GaussianBlur
```
Comment (MylesBartlett) [inline conduit/models/self_supervised/dino/transforms.py:13]: Still think this should stay in moco

File: conduit/models/self_supervised/moco/transforms.py (hunk 3 above)
Comment (MylesBartlett) [reply conduit/models/self_supervised/moco/transforms.py:21]: Except for the fact that the implementation and default parameterisation matches that in the official MoCo implementation - different libraries implement the Gaussian kernel in substantially different ways (in the way that the radius is computed, for instance)

**Review by MylesBartlett — APPROVED** [review]
